F/FastJet/build_tarballs.jl

```diff
@@ -81,4 +81,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version=v"8")
```

Comment:
Not sure we need a version as high as 8, can maybe try to use something lower?  Also, can we require Julia v1.6 (`julia_compat="1.6"`) so we can build for experimental platforms?

Reply:
5 is too low, but 7 works.